testing/loki_alerts/Couchbase-Server/CB90071-compactionFailure/debug.log

```diff
@@ -0,0 +1,2 @@
+[user:error,2021-11-15T14:28:59.661Z,ns_1@ipaddress:compaction_daemon:ensure_can_db_compact:1024]Cannot compact database `database name`: the estimated necessary disk space is about 3498444 bytes but the currently available disk space is 0 bytes.
```

Comment:
Is "database name" meant to be the name of a bucket?

Reply:
Yeah, it had a customers info there so I just replaced it

Reply:
You may want to replace `ipaddress` with a valid (though made-up) IP, and `database name` with a valid (though made-up) bucket name (namely, they can't contain spaces) - just to keep the logs realistic.

(this is why I recommend you use logs from your own reproduction ;) )